java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -549,24 +581,27 @@ abstract class AbstractArcHost(
     override suspend fun isHostForParticle(particle: Plan.Particle) =
         registeredParticles().contains(ParticleIdentifier.from(particle.location))
 
+    @Deprecated("Provide a Configuration with a HandleManagerProvider")
     /**
      * Return an instance of [EntityHandleManager] to be used to create [Handle]s.
      */
     open fun entityHandleManager(arcId: String) = EntityHandleManager(
         arcId,
         hostId,
         platformTime,
-        schedulerProvider(arcId),
+        schedulerProvider!!(arcId),
```

Comment:
I think we decided !! is naughty. 

better would be requireNotNull(schedulerProvider) { errorMessage }.invoke(arcId) I think